client/app/connect/connect.controller.js

```diff
@@ -2,7 +2,7 @@ angular.module('OpenEXP')
     .controller('ConnectCtrl', ['$scope','boardFactory', 'simulatorFactory', ($scope, boardFactory, simulatorFactory) => {
         var _ = require('lodash');
 
-        $scope.connect = boardFactory.connect;
+        $scope.connect = simulatorFactory.connect;
```

Comment:
so that @tashoecraft can stream in data without a board...eventually it will eliminated, but good for now.  On the next PR I have queued up, I have this part cleaned up quite a bit